portfolio/src/main/webapp/pets.html

```diff
@@ -21,30 +21,35 @@ <h1>Winnie Wan</h1>
       </ul>
     </nav>
 
-    <div id="content">
-      <h1>My Guinea Pigs: Cookie and Milky</h1>
-      <img src="images/pets/gp1.jpeg" alt="guineapig"/>
-      <img src="images/pets/gp2.jpeg" alt="guineapig"/>
-      <img src="images/pets/gp6.jpg" alt="guineapig"/>
-      <img src="images/pets/gp3.jpeg" alt="guineapig"/>
-      <img src="images/pets/gp7.jpg" alt="guineapig"/>
-      <img src="images/pets/gp4.jpeg" alt="guineapig"/>
-      <img src="images/pets/gp5.jpeg" alt="guineapig"/>
+    <div class="content">
+      <h2>My Guinea Pigs: Cookie and Milky :)</h2>
+      <div class="images">
+        <img src="images/pets/gp1.jpeg" class="image" alt="guineapig"/>
+        <img src="images/pets/gp2.jpeg" class="image" alt="guineapig"/>
+        <img src="images/pets/gp6.jpg" class="image" alt="guineapig"/>
+        <img src="images/pets/gp3.jpeg" class="image" alt="guineapig"/>
+        <img src="images/pets/gp7.jpg" class="image" alt="guineapig"/>
+        <img src="images/pets/gp4.jpeg" class="image" alt="guineapig"/>
+        <img src="images/pets/gp5.jpeg" class="image" alt="guineapig"/>
```

Comment:
any reason the images aren't in numerical order here?

Reply:
I was testing some things out. It's fixed.